charts/argo-templates/templates/ingest-file.yaml

```diff
@@ -19,26 +19,10 @@ spec:
           - name: sa-secret-key
       dag:
         tasks:
-          # 1) Make sure that something doesn't already exist at the target path
```

Comment:
Glad we removed these numbered steps - hopefully will be less confusing to extend/change things in the future.